content/en/docs/reference/access-authn-authz/validating-admission-policy.md

```diff
@@ -207,15 +208,17 @@ apiVersion: rules.example.com/v1
 kind: ReplicaLimit
 metadata:
   name: "replica-limit-test.example.com"
+  namesapce: "default"
 maxReplicas: 3
 ```
 
-This policy parameter resource limits deployments to a max of 3 replicas in all namespaces in the
-test environment. An admission policy may have multiple bindings. To bind all other environments
+This policy parameter resource limits deployments to a max of 3 replicas.
+
+An admission policy may have multiple bindings. To bind all other environments
 environment to have a maxReplicas limit of 100, create another ValidatingAdmissionPolicyBinding:
 
 ```yaml
```

Comment:
Could this use a [`codenew`](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#source-code-files) shortcode?